openshift/ci-operator/generate-ci-config.sh

```diff
@@ -3,9 +3,14 @@
 branch=${1-'knative-v0.19.1'}
 openshift=${2-'4.6'}
 promotion_disabled=${3-false}
+generate_continuous=true
 
 if [[ "$branch" == "knative-next" ]]; then
-    branch="knative-nightly"
+  branch="knative-nightly"
```

Comment:
why set to knative-nightly there, instead of keeping it -next?

Reply:
yeah that's historic stuff, let me see if I clean it up more. probably just gonna need another variable with 'knative-nightly'

Reply:
not even needed, I'll remove that line.